subscribers/logs/logger.tests/LoggerTests.cs

```diff
@@ -0,0 +1,94 @@
+using System;
+using Xunit;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Moq;
+using Newtonsoft.Json;
+using Microsoft.Data.Sqlite;
+using Dta.Marketplace.Subscribers.Logger.Worker;
+using Amazon.SQS.Model;
+
+using Microsoft.EntityFrameworkCore;
+
+namespace Dta.Marketplace.Subscribers.Logger.Worker.Logger.Tests {
+
+    public class LoggerTests {
+
+        [Fact]
+        public void Can_log_information_message() {

```

Comment:
The naming of methods / functions should be Pascal case.  Check [Microsoft's conventions](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions).
```suggestion
        public void CanLogInformationMessage() {
```

Reply:
fixed to add the right c# naming convengtion